docs/runbook/maintain-cardano-min-versions-file/README.md

```diff
@@ -0,0 +1,26 @@
+#  Cardano minimum versions for Signers
+
+## Introduction
+
+The [cardano-min-versions.json](../../../cardano-min-versions.json) file, which specifies the minimum supported versions of the Cardano node for Mithril signers
+This ensures compatibility and functionality across different networks.
+
+## Maintaining the file
+
+To maintain compatibility when a new Cardano node version is released, follow these steps:
+
+### Test for compatibility:
+- Create a pull request to test the new Cardano node version in the devnet using CI end-to-end tests.
```

Comment:
Can you explain the modifications to do in detail?

Reply:
You are talking steps?:
- update the default value of the `cardano_node_version` clap argument in e2e
- update `CARDANO_NODE_VERSION` argument in the Aggregator and Signer Dockerfiles
- update of the `e2e` job in the `ci.yml` file (adding the new cardano node version in the list of tested environments)

Reply:
Yes, but the most important is the last one which we use to test compatibility :+1:

Reply:
Done ✅